crates/re_sdk/src/web_viewer.rs

```diff
@@ -21,8 +21,8 @@ struct WebViewerSink {
     /// Sender to send messages to the [`re_ws_comms::RerunServer`]
     sender: re_smart_channel::Sender<LogMsg>,
 
-    /// Handle to keep the [`re_ws_comms::RerunServer`] alive
-    _rerun_server: RerunServerHandle,
+    /// Rerun websocket server.
+    _rerun_server: RerunServer,
```

Comment:
note that there's no distinction between `RerunServer` and `RerunServerHandle` now